index.js

```diff
@@ -92,6 +93,15 @@ Parser.prototype._transform = function(file, encoding, done) {
 
 
 
+    // we find the relative path
+    // =========================
+    var relativeFilePath = path.relative(__dirname, this.base);
+    if (file.path) {
```

Comment:
I would prefer setting the `file.path` in the tests rather than having "test specific" code in the main script.